app/server.py

```diff
@@ -39,11 +35,11 @@ def search(search_engine):
         num = request.args.get('num') or 10
         count = int(num)
         qformat = request.args.get('format') or 'json'
-        if qformat not in ['json', 'xml']:
+        if qformat not in ('json', 'xml'):
             abort(400, 'Not Found - undefined format')
 
         engine = search_engine
-        if engine not in ['google', 'bing', 'duckduckgo', 'yahoo', 'ask']:
+        if engine not in ('google', 'bing', 'duckduckgo', 'yahoo', 'ask'):
```

Comment:
Same

Reply:
As it says in the comments above, tuples are more compact and unmodifiable than lists.